source/cellular_pkthandler.c

```diff
@@ -93,7 +93,6 @@ static CellularPktStatus_t _convertAndQueueRespPacket( CellularContext_t * pCont
     if( ( pBuf != NULL ) )
     {
         pAtResp = ( const CellularATCommandResponse_t * ) pBuf;
-        PlatformMutex_Lock( &pContext->PktRespMutex );
```

Comment:
Why do we need to remove the lock here?
We still need to read pktRespCB in pContext. Don't we need to protect this part?

Reply:
Yes, the packet handler callback is called by pktio thread. Pktio thread acquires the lock already.